### PR TITLE
First implementation of Eclipse Marketplace dp version check

### PR DESCRIPTION
**Pull request by nicolatimeus**
See #1032

Signed-off-by: nicolatimeus <nicola.timeus@eurotech.com>

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Packages/PackagesPanelUi.java [hunk 1]
```diff
@@ -122,6 +127,24 @@
     @UiField
     Well marketplaceInstallWell;
 
+    @UiField
```
Comment (MMaiero) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Packages/PackagesPanelUi.java:124]: Can these UIField be private?
Maybe also marketplaceDescriptor can be private aswell

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Packages/PackagesPanelUi.java [hunk 2]
```diff
@@ -448,48 +482,57 @@ public void onSuccess(GwtXSRFToken token) {
...
+                                }
+                            }
+                        });
             }
         });
     }
 
-    private void initModal() {
+    private void initModals() {
         this.uploadErrorModal.setTitle(MSGS.warning());
```
Comment (MMaiero) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Packages/PackagesPanelUi.java]: let's add two private methods for respective modals initialization.

**Comment by MMaiero** [comment]
@nicolatimeus I've rebased the branch and made minor changes. Please have a look.

**Comment by nicolatimeus** [comment]
The changes seem fine

**Comment by MMaiero** [comment]
Ok, Merging then.

**Pull request merged**